.github/workflows/merge-content.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     steps:
       - name: Checkout full repo
-        uses: actions/checkout@v4
+        uses: actions/checkout@v5
```

Comment:
Can we use this everywhere? It looks like in most places we're using `v4`

Reply:
It looks like Renovatebot can and should automatically be configured to look for version updates, but that would be a good thing for us to verify.